CONTRACT-TESTS.md

```diff
@@ -0,0 +1,20 @@
+# API Contract Tests
+
+It's highly recommended to use [Virtualenv](https://virtualenv.pypa.io/en/latest/)
+in order to have an isolated environment.
+
+From the root directory of the project
+
+1. `virtualenv venv`
+2. `source venv/bin/activate` to turn on the virtualenv
+2. `./venv/bin/pip install -r test-requirements.txt`
```

Comment:
Oh wait, can these tests run in Python 3? I assumed, but then I didn't notice any Python2-only libraries in the requirements below. If that's the case, definitely defer to the standard setup docs for setting up an environment.